examples/azureiot_esp32spi/azureiot_central_commands.py

```diff
@@ -8,7 +8,7 @@
 import neopixel
 import rtc
 from adafruit_esp32spi import adafruit_esp32spi, adafruit_esp32spi_wifimanager
-import adafruit_esp32spi.adafruit_esp32spi_socket as socket
+import adafruit_connection_manager
```

Comment:
Could you add the connection manager package to the [Requirements](https://github.com/adafruit/Adafruit_CircuitPython_AzureIoT/blob/main/requirements.txt) file?

Reply:
should I get this fully current? It also needs minimqtt, logging, and binascii.

And there and in the README - Dependencies?

Reply:
@dhalbert do you know what would be correct here? I think it's just adding the 3 real dependencies to the readme, no?

And minimqtt requires `ConnectionManager`, so it would add that with circup, yes?

Reply:
@brentru looking, I see some stuff is in there already (missed it before). I actually think we should:
1. remove `adafruit-circuitpython-requests` from the requirements, and all the places it states it (since it's not actually used (at least not directly)
2. remove `adafruit-circuitpython-connectionmanager` from the Dependencies since it would be installed by mimqtt

I did test this via pip and connectionmanager is installed

Reply:
That sounds OK to me, especially if connection manager is already installed by MiniMQTT (underlying layer for this lib)

Reply:
Dependent libraries are only installed if one is using pip or circup or a Project bundle. If someone is doing this manually, they would need to know all the dependecies, though they could look at the direct dependencies and do their own manual tree walk. @FoamyGuy do you know if we have a policy of leaving out indirect dependencies in the README files?

It's true the indirect dependencies can change. The phrasing could be changed from "This driver depends on:" to "This driver depends on these libraries and their dependencies:" or similar. _I am not suggesting we make that change now, to this library,_ just that we need to understand whether we have a policy, whether it's consistent across libraries, etc.

Reply:
Fair. I think trying to keep track of indirect dependency changes, seems like a pain, and having the library point directly to what it needs. But happy to add connectionmanager back in (it didn't need requests though)

Reply:
We don't need to resolve this before merging.